### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Screen Reader: NVDA/Narrator is not reading the name for "Menu" item present near Enable/Disable and "Uninstall" buttons

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2674]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yTCS;#BM-Azure Tools for VS Code-Win-Feb2021; #AILimited; #DesktopApp; #Screen Reader; #WCAG4.1.2; #Name, role, value; #NVDA; #Win10; #Narrator;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Turn on NVDA/NVDA
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Tab to all controls and verify how NVDA/Narrator is reading "Menu" item present near Enable/Disable and "Uninstall" buttons

### Actual Result:
NVDA/Narrator is not reading the name for "Menu" item present near Enable/Disable and "Uninstall" buttons

### Expected Result:
NVDA/Narrator should read proper name for "Menu" item present near Enable/Disable and "Uninstall" buttons.

### User Impact:
Assistive technology users will not understand the complete information of controls if screen reader is not announcing proper names for them

### Attachment for Reference:

![2674_Narrator issue](https://user-images.githubusercontent.com/38523250/108310151-e1cb3e00-71d8-11eb-844f-243ec5fafae0.png)
[2674_NVDA issue.zip](https://github.com/microsoft/vscode-azurefunctions/files/6020036/2674_NVDA.issue.zip)

**Comment by ejizba** [comment]
Our extension does not own or control the marketplace. This would reproduce for any VS Code extension and should be filed directly on VS Code instead: https://github.com/microsoft/vscode/issues
